src/Bridges/Constraint/relentr_to_exp.jl

```diff
@@ -0,0 +1,116 @@
+    func = MOIU.zero_with_output_dimension(G, 1 + 2 * length(bridge.y))
+    MOIU.operate_output_index!(+, T, 1, func, MOI.get(model, MOI.ConstraintFunction(), bridge.ge_index))
+    w_start = 1 + length(bridge.y)
+    for i in eachindex(bridge.y)
+        exp_func_i = MOIU.eachscalar(MOI.get(model, MOI.ConstraintFunction(), bridge.exp_indices[i]))
+        MOIU.operate_output_index!(-, T, 1, func, exp_func_i[1])
+        MOIU.operate_output_index!(+, T, 1 + i, func, exp_func_i[3])
+        MOIU.operate_output_index!(+, T, w_start + i, func, exp_func_i[2])
+    end
+    return MOIU.convert_approx(H, MOIU.remove_variable(func, bridge.y))
+end
+MOI.get(model::MOI.ModelLike, ::MOI.ConstraintSet, bridge::RelativeEntropyBridge) = MOI.RelativeEntropyCone(1 + 2 * length(bridge.y))
+MOI.supports(::MOI.ModelLike, ::Union{MOI.ConstraintPrimalStart, MOI.ConstraintDualStart}, ::Type{<:RelativeEntropyBridge}) = true
+function MOI.get(model::MOI.ModelLike, attr::Union{MOI.ConstraintPrimal, MOI.ConstraintPrimalStart}, bridge::RelativeEntropyBridge{T}) where T
+    primal = zeros(T, 1 + 2 * length(bridge.y))
+    primal[1] = MOI.get(model, attr, bridge.ge_index)
+    w_start = 1 + length(bridge.y)
+    for i in eachindex(bridge.y)
+        exp_primal_i = MOI.get(model, attr, bridge.exp_indices[i])
+        primal[1] -= exp_primal_i[1]
+        primal[1 + i] = exp_primal_i[3]
+        primal[w_start + i] = exp_primal_i[2]
+    end
+    return primal
+end
+function MOI.set(model::MOI.ModelLike, attr::MOI.ConstraintPrimalStart, bridge::RelativeEntropyBridge{T}, value) where T
+    v_dim = length(bridge.y)
+    v_value = value[2:(v_dim + 1)]
+    w_value = value[(v_dim + 2):end]
+    y_value = [w_i * log(w_i / v_i) for (v_i, w_i) in zip(v_value, w_value)]
+    MOI.set(model, attr, bridge.ge_index, value[1] - reduce(+, y_value, init=zero(T)))
+    for i in 1:v_dim
+        MOI.set(model, attr, bridge.exp_indices[i], [-y_value[i], w_value[i], v_value[i]])
+        MOI.set(model, MOI.VariablePrimalStart(), bridge.y[i], y_value[i])
+    end
+    return
+end
+# Given a is dual on u - sum(y) >= 0 and (b_i, c_i, d_i) is dual on (-y_i, w_i, v_i)
+# in ExponentialCone, the dual on (u, v, w) in RelativeEntropyCone is (a, d_i, c_i).
+function MOI.get(model::MOI.ModelLike, attr::Union{MOI.ConstraintDual, MOI.ConstraintDualStart}, bridge::RelativeEntropyBridge{T}) where {T}
+    dual = zeros(T, 1 + 2 * length(bridge.y))
+    dual[1] = MOI.get(model, attr, bridge.ge_index)[1]
+    w_start = 1 + length(bridge.y)
+    for i in eachindex(bridge.y)
+        exp_dual_i = MOI.get(model, attr, bridge.exp_indices[i])
+        dual[1 + i] = exp_dual_i[3]
+        dual[w_start + i] = exp_dual_i[2]
+    end
+    return dual
+end
+# Given constraint dual start of (u, v, w), constraint dual on GreaterThan is u
+# and on exponential cone constraint i is (r_i, w_i, v_i), but since y_i is free,
+# its dual is 0, so we have -r_i + value[1] == 0 hence r_i = value[1].
```

Comment:
Add that r_i could also be computed using the Lambert W function but it we choose to use value[1] instead

Reply:
Otherwise, when we read this later, we might wonder why we are not computing it the other way

Reply:
Done, good idea